kyo-core/shared/src/main/scala/kyo/KyoApp.scala

```diff
@@ -30,21 +25,28 @@ object KyoApp:
       *   The effect type used by the application.
       */
     abstract class Base[S]:
+        protected def timeout: Duration
 
-        protected def handle[A: Flat](v: A < S)(using Frame): Unit
+        protected def handle[A: Flat](v: A < S)(using Frame): A < (Async & Abort[Throwable])
 
         final protected def args: Array[String] = _args
 
         private var _args: Array[String] = null
 
-        private val initCode = new ListBuffer[() => Unit]
+        protected var maybeProc: Maybe[() => Unit] = Absent
```

Comment:
Can this be `Iterable[() => Unit]`? Or `IterableOnce`? I think the changes in this PR break multiple runs on JVM.